src/Console/Commands/OptimizeCommand.php

```diff
@@ -26,7 +26,7 @@ final class OptimizeCommand extends Command
     /**
      * {@inheritdoc}
      */
-    protected $signature = 'scout:optimize {searchable? : The name of the searchable}';
+    protected $signature = 'scout:optimize {searchable? : The name of the searchable} {--prefix=}';
```

Comment:
You should add a description on the prefix, explaining what it means?

Reply:
It's not easy finding the right way to describe it, but I was thinking: `A custom prefix to create a settings file with`.

Reply:
"A custom `scout.prefix` configuration key."